src/types/Recipe.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import BaseValidator from './base.js';
+
+export default class RecipeValidator extends BaseValidator {
+  getConditions() {
+    const conditions = [
+      this.required('image', 'url'),
+      this.required('name', 'string'),
+
+      this.recommended('aggregateRating'),
+      this.recommended('author'),
+      this.recommended('datePublished', 'date'),
+      this.recommended('description', 'string'),
+      this.recommended('keywords', 'string'),
+      this.recommended('recipeCategory', 'string'),
+      this.recommended('recipeCuisine', 'string'),
+      this.recommended('recipeIngredient'),
+      this.recommended('recipeInstructions'),
+      this.validateNutritionAndYield,
+      this.validateCookTime,
+      this.recommended('video'),
+    ];
+    return conditions.map((c) => c.bind(this));
+  }
+  validTimeFormat(time) {
```

Comment:
There are some other entities that use this format (e.g. VideoObject). So it would make sense to add this as type check in the base object.